universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -50,6 +50,9 @@ public static Builder builder() {
   /** An external link to a page containing more details for this program. */
   public abstract String externalLink();
 
+  /** The program's display mode. */
+  public abstract String displayMode();
```

Comment:
question (blocking): why `string` here rather than `DisplayMode`? it looks like most of the code ends up dealing directly with strings rather than enums for this value which sidesteps helpful type checking.

Reply:
Done!